src/main/java/uk/gov/digital/ho/hocs/audit/export/DataExportResource.java

```diff
@@ -164,6 +165,13 @@ void getCustomDataExport(HttpServletResponse response,
 
     }
 
+    @PostMapping(value = "/export/custom/{viewName}/refresh")
+    public @ResponseBody void refreshMaterialisedView(HttpServletResponse response, @PathVariable("viewName") final String viewName) {
+        int result = customExportService.refreshMaterialisedView(viewName);
+        System.out.println(result);
+        response.setStatus(200);
```

Comment:
Make sure this only sends 200 if it works. If it fails and the cronjob needs to retry later, it needs to send a 5xx error code.

Reply:
I've changed this and got rid of the uncaught exception.

Reply:
This has been wrapped correctly and returns a 500 on error.